src/hcpp-runtime.c

```diff
@@ -45,16 +45,24 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include "hcpp-finish.h"
 #include "hcpp-hpt.h"
 #include "hcupc-support.h"
+#include "hcpp-cuda.h"
 
 // #define VERBOSE
 
+static const int communication_worker_id = 1;
```

Comment:
Does this mean, the communication worker id = 1 and not 0 ? If yes, then this would affect HabaneroUPC++ significantly